uaclient/cli.py

```diff
@@ -896,6 +973,15 @@ def action_config_unset(args, *, cfg, **kwargs):
                 ", ".join(config.UA_CONFIGURABLE_KEYS)
             )
         )
+    ua_scoped_proxy_options = ("ua_apt_http_proxy", "ua_apt_https_proxy")
+    global_scoped_proxy_options = (
+        "global_apt_http_proxy",
+        "global_apt_https_proxy",
+    )
+    deprecated_global_scoped_proxy_options = (
+        "apt_http_proxy",
+        "apt_https_proxy",
+    )
```

Comment:
We are defining those constant in two different functions, I think we can move them somewhere else, maybe `config` or `defaults`

Reply:
I think `config` is better, I see that `default` is a bit global.